src/patterns/phone-numbers/index.njk

```diff
@@ -0,0 +1,119 @@
+---
+title: Phone numbers
+group: Ask users for...
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/lists/_macro.njk" import onsList %}
+
+
+How to ask users for a phone number.
+
+{{
+    patternlibExample({"path": "components/input/examples/telephone/index.njk"})
+}}
+
+## When to use this pattern
+Only ask users for a phone number when your service has a genuine need for this information. For example, to send an access code by text message. 
+
+## How to use this pattern
+You should always explain why you need the user's phone number, letting them know if you will contact them and when.
+
+Use an [input component](/components/input) and set the input type to `tel` to bring up the correct keypad on touch devices.
```

Comment:
Yes for line 21 but it's already there in the first bullet point so I'd just add a second one about contact:

- tell users why you need a phone number and what it will be used for
- let users know if you will contact them and when
- make it clear what type of telephone number you need
- make sure you allow the user to enter a phone number in a familiar format
- help users enter a phone number in the correct format

But I think for line 23, the bit about the component to use doesn't normally go in the bullet list as it's a specific instruction rather than the rest of the list which is general guidance